pybotters/models/gmocoin.py

```diff
@@ -0,0 +1,616 @@
+    if isinstance(x, str):
+        return datetime.strptime(x, "%Y-%m-%dT%H:%M:%S.%f%z")
+    else:
+        raise ValueError(f'x only support str, but {type(x)} passed.')
+
+
+class ApiType(Enum):
+    """
+    API 区分
+    """
+
+    Public = auto()
+    Private = auto()
+
+
+class Channel(Enum):
+    """
+    WebSocket API チャンネル
+    """
+
+    # Public
+    TICKER = auto()
+    ORDER_BOOKS = auto()
+    TRADES = auto()
+    # Private
+    EXECUTION_EVENTS = auto()
+    ORDER_EVENTS = auto()
+    POSITION_EVENTS = auto()
+    POSITION_SUMMARY_EVENTS = auto()
+
+    @staticmethod
+    def from_str(name: str) -> "Channel":
+        if not hasattr(Channel, "_table"):
+            Channel._table = {
+                "ticker": Channel.TICKER,
+                "orderbooks": Channel.ORDER_BOOKS,
+                "trades": Channel.TRADES,
+                "executionEvents": Channel.EXECUTION_EVENTS,
+                "orderEvents": Channel.ORDER_EVENTS,
+                "positionEvents": Channel.POSITION_EVENTS,
+                "positionSummaryEvents": Channel.POSITION_SUMMARY_EVENTS,
+            }
+        return Channel._table[name]
+
+
+class MessageType(Enum):
+    """
+    メッセージタイプ
+    """
+
+    NONE = auto()
+    ER = auto()
+    NOR = auto()
+    ROR = auto()
+    COR = auto()
+    OPR = auto()
+    UPR = auto()
+    ULR = auto()
+    CPR = auto()
+    INIT = auto()
+    UPDATE = auto()
+    PERIODIC = auto()
+
+
+class Symbol(Enum):
+    """
+    取り扱い銘柄
+    """
+
+    BTC = auto()
+    ETH = auto()
+    BCH = auto()
+    LTC = auto()
+    XRP = auto()
+    BTC_JPY = auto()
+    ETH_JPY = auto()
+    BCH_JPY = auto()
+    LTC_JPY = auto()
+    XRP_JPY = auto()
+
+
+class OrderSide(Enum):
+    """
+    売買区分
+    """
+
+    BUY = auto()
+    SELL = auto()
+
+
+class ExecutionType(Enum):
+    """
+    注文タイプ
+    """
+
+    MARKET = auto()
+    LIMIT = auto()
+    STOP = auto()
+
+
+class TimeInForce(Enum):
+    """
+    執行数量条件
+    """
+
+    FAK = auto()
+    FAS = auto()
+    FOK = auto()
+    SOK = auto()
+
+
+class SettleType(Enum):
+    """
+    決済区分
+    """
+
+    OPEN = auto()
+    CLOSE = auto()
+    LOSS_CUT = auto()
+
+
+class OrderType(Enum):
+    """
+    取引区分
+    """
+
+    NORMAL = auto()
+    LOSSCUT = auto()
+
+
+class OrderStatus(Enum):
+    """
+    注文ステータス
+    """
+
+    WAITING = auto()
+    ORDERED = auto()
+    MODIFYING = auto()
+    CANCELLING = auto()
+    CANCELED = auto()
+    EXECUTED = auto()
+    EXPIRED = auto()
+
+
+class CancelType(Enum):
+    """
+    取消区分
+    """
+
+    NONE = auto()
+    USER = auto()
+    POSITION_LOSSCUT = auto()
+    INSUFFICIENT_BALANCE = auto()
+    INSUFFICIENT_MARGIN = auto()
+    ACCOUNT_LOSSCUT = auto()
+    MARGIN_CALL = auto()
+    MARGIN_CALL_LOSSCUT = auto()
+    EXPIRED_FAK = auto()
+    EXPIRED_FOK = auto()
+    EXPIRED_SOK = auto()
+    CLOSED_ORDER = auto()
+    SOK_TAKER = auto()
+    PRICE_LIMIT = auto()
+
+
+@dataclass
+class Ticker(TypedDict):
+    ask: int
```

Comment:
https://github.com/MtkN1/pybotters/pull/74/files#diff-70d03f7374b991e4ed0cf280951f8c425f5b3275616ac1d3c0c1a61872916583R395 に付随
```suggestion
    ask: Decimal
```